index.bs

```diff
@@ -739,6 +739,11 @@ interface MLOperand {};
 </dl>
 </div>
 
+To get the <dfn for="MLOperand">rank</dfn> of an {{MLOperand}} |operand|, run the following steps:
+<div algorithm=rank class=algorithm-steps>
+    1. Return the size of |operand|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}}.
+</div>
+
```

Comment:
But then one could say if the MLOperandDescriptor is changed, then the link between that and the internal slot is lost, so it would be even harder to track the changes...
It is quite common to have internal slots of dictionary types defined in the spec.
BTW I have actually started with what you suggest, but ended up with this since it's simpler.
@huningxin any opinion?